app/models/topic.rb

```diff
@@ -41,8 +41,15 @@ class Topic < ActiveRecord::Base
 
   acts_as_readable
 
-  def self.recent_topics
-    find_by_sql %q{
+  def self.recent_topics user=nil
```

Comment:
Use def with parentheses when there are parameters.<br>Surrounding space missing in default value assignment.